tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -21,6 +21,54 @@
 
 import pytest
 
+import robottelo.constants as constants
+from robottelo.config import settings
+
+
+@pytest.mark.e2e
+def test_positive_install_sat_with_katello_certs(certs_data, sat_ready_rhel):
+    """Install Satellite with custom certs.
+
+    :id: 47e3a57f-d7a2-40d2-bbc7-d1bb3d79a7e1
+
+    :steps:
+
+        1. Generate the custom certs on RHEL machine
+        2. Install satellite with custom certs
+        3. Assert output does not report SSL certificate error
+        4. Assert all services are running
+
+
+    :expectedresults: Satellite should be installed using the custom certs.
+
+    :CaseAutomation: Automated
+    """
+    version = sat_ready_rhel.os_version.major
+    sat_ready_rhel.download_repofile(product='satellite', release=settings.server.version.release)
+    sat_ready_rhel.register_to_cdn()
+    for repo in getattr(constants, f"OHSNAP_RHEL{version}_REPOS"):
+        sat_ready_rhel.enable_repo(repo, force=True)
```

Comment:
As register_to_cdn auto-attaches the baseos and appstream repos, so we might not need this 
```suggestion

```